.circleci/config.yml

```diff
@@ -1395,91 +1396,256 @@ jobs:
 
+  build-windows-package:
+    description: "Build windows MSI installer."
+    executor:
+      name: win/default
+      size: "medium"
+      shell: powershell.exe
+    steps:
+      - checkout
+      - restore_cache:
+          name: Restore Pyinstaller Cache
+          key: windows_python_installer-python-3.7.7-{{ .Branch }}
+
+      - run:
+          name: Install python.
+          command: |
+            $ProgressPreference = "SilentlyContinue"
+            mkdir python_installer -Force
+            if (!(Test-Path .\python_installer\python-3.7.7-amd64.exe -PathType Leaf)) {
+                wget -O .\python_installer\python-3.7.7-amd64.exe https://www.python.org/ftp/python/3.7.7/python-3.7.7-amd64.exe
+            }
+            .\python_installer\python-3.7.7-amd64.exe /install /quiet TargetDir=c:\Python37 Include_tcltk=0
+
+      - save_cache:
+          name: Save Pyinstaller Cache
+          key: windows_python_installer-python-3.7.7-{{ .Branch }}
+          paths:
+            - "python_installer"
+
+      - restore_cache:
+          name: Restore Python Dependencies Cache
+          key: windows_python_installer-python-3.7.7-{{ .Branch }}-py-deps-cache
+
+      - run:
+          name: Build MSI package
+          command: |
+             .\win32\build.ps1 c:\Python37
+              mkdir package
+              mv .\ScalyrAgentInstaller-*.msi package\ScalyrAgentInstaller.msi
+
+      - save_cache:
+          name: Save Python Dependencies Cache
+          key: windows_python_installer-python-3.7.7-{{ .Branch }}-py-deps-cache
+          paths:
+            - C:\Users\circleci\AppData\Local\pip\Cache
+
+      - persist_to_workspace:
+          root: package
+          paths: ScalyrAgentInstaller.msi
+
+      - store_artifacts:
+          path: package
+
+  build-linux-packages:
+    description: "Build deb and rpm packages."
+    working_directory: ~/scalyr-agent-2
+    docker:
+        - image: circleci/python:3.6-jessie
+    steps:
+      - checkout
+      - restore_cache:
+          key: deps2-tox-{{ .Branch }}-3.6-venv-{{ checksum "dev-requirements.txt" }}
+
+      - run:
+          name: Install System Dependencies
+          command: |
+            sudo apt-key update
+            sudo apt update
+            sudo apt install -y ruby ruby-dev rubygems rpm build-essential --force-yes
+
+            # TODO: looks like fpm upgraded its 'ffi' dependency version, which is now requres newer version of ruby.
+            # as a temporaty solution explicitly specify ffi version.
+            sudo gem install --no-document ffi:1.12.2 fpm
+
+      - run:
+          name: Build deb and rpm packages
+          command: |
+            mkdir artifacts
+            mkdir package
+
+            ls
+            pushd artifacts
+
+            # update agent version to avoid version conflict with current scalyr repository package.
+            #echo "$(cat VERSION).1" > VERSION
+
+            python ../build_package.py deb
```

Comment:
To make it easier to distinguish packages, I think it would be a good idea to include git revision in the filename (I believe we did the same thing with previous build package job).

Reply:
Basically same thing as we do in one of the Dockerfiles - we append git commit sha to the version.

And we should also bump version in ``VERSION`` file so we can distinguish between different versions on upgrade tests.

Right now if you look at the script output, same version will be printed in both cases (pre and after the upgrade) which is confusing and incorrect.

Reply:
And to give some context - right now we use the same version for current / from and to package version which means our upgrade tests are not 100% correct and robust - the version is the same in both cases so we can't correctly assert if the upgrade was successful or not.

Anyway, I pushed some changes which I believe should address that.

Before: https://41733-23852161-gh.circle-artifacts.com/0/outputs/ubuntu1804.log (as you can see the version pre and post upgrade is the same)
After: https://41741-23852161-gh.circle-artifacts.com/0/outputs/ubuntu1804.log (version after the upgrade is different)